app/webpack/reactjs/components/art_page.tsx

```diff
@@ -36,7 +32,11 @@ export const ArtPage: FC<ArtPageProps> = ({ artistId, initialArtPieceId }) => {
 
   const cardClasses =
     "pure-u-1-1 pure-u-sm-1-1 pure-u-md-1-2 pure-u-lg-1-4 pure-u-xl-1-4";
-  return Object.entries(artPieces).map(([id, artPiece]) => (
-    <ArtCard key={id} artPiece={artPiece} classes={cardClasses} />
-  ));
+  return (
```

Comment:
Initially i thought keyed by index was better, but I forgot, the order of the array is important because that is the order as specified by an artist.  Artists can arrange their art in order.  So we need to use the array.